src/app/services/EloCalculationService.ts

```diff
@@ -0,0 +1,106 @@
+import { Injectable } from '@angular/core';
+import { Player } from '../jscaip/Player';
+import { Utils } from '../utils/utils';
+import { EloInfo } from '../domain/EloInfo';
+
+export interface EloInfoPair {
```

Comment:
avoid explicit names, prefer `type EloInfoPair = [EloInfo, EloInfo]`, qu'on puisse génériquement itérer dessus

Reply:
c'bin bin mieux comme ço lo !